packages/vite/types/clean-css.d.ts

```diff
@@ -453,7 +453,7 @@ export namespace CleanCSS {
       /**
        * denotes a number of /*! ... * / comments preserved; defaults to `all`
        */
-      specialComments?: string
+      specialComments?: 'all' | 1 | 0
```

Comment:
**question:** Are `'1' | '0'` also possible values?

Reply:
Good catch. I updated it to allow that too